CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/")
 include(FeatureSummary)
 include(CMakeDependentOption)
 
+# C&C Renegade does not use import modules
+set(CMAKE_CXX_SCAN_FOR_MODULES OFF)
```

Comment:
Can we move this to compilers.cmake which is where all the compiler configuration stuff has been put?